grabbit/core.py

```diff
@@ -407,6 +549,9 @@ def get(self, return_type='tuple', target=None, extensions=None,
             result = [r.as_named_tuple() for r in result]
             return natural_sort(result, field='filename')
 
+        if return_type == 'File':
```

Comment:
Oh, nice--thanks for adding that here! I think the file/File thing might be confusing though, so maybe we should go with 'object' (or maybe `startswith('obj')`) for the new option. That's a bit clunky, but since the more sensible approach (having 'filename' and 'File') would be a breaking change, I think it's probably preferable.